src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,43 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String results = getStatisticValuesFromInputFile(fromFileName);
+        writeStatisticToFile(results, toFileName);
+    }
+
+    private String getStatisticValuesFromInputFile(String inputFileName) {
+        int buyAmount = 0;
+        int supplyAmount = 0;
+        try (BufferedReader bfReader = new BufferedReader(new FileReader(inputFileName))) {
+            String str = bfReader.readLine();
+            while (str != null) {
+                if (str.startsWith("b")) {
+                    buyAmount += Integer.parseInt(str.replaceFirst("\\w+,", ""));
+                } else {
+                    supplyAmount += Integer.parseInt(str.replaceFirst("\\w+,", ""));
+                }
+                str = bfReader.readLine();
+            }
+        } catch (IOException e) {
```

Comment:
Performing business logic during file reading make connection to file to be opened for longer period of time what is not desirable.
Please create separate method to read file content and then proceed with calculations in separate method